bazel/repository_locations.bzl

```diff
@@ -1204,4 +1204,16 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2022-03-02",
         cpe = "N/A",
     ),
+    dlb = dict(
```

Comment:
im wondering if we should use a longer name for this dependency - something that gives more indication as to what it is/ where its from

Reply:
How about `intel_dlb`?